tests/test_jit.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Unit tests for exporting models via TorchScript (i.e. JIT compilation).
+"""
+
+import os
+import regex
+import unittest
+
+import parlai.utils.testing as testing_utils
+from parlai.agents.repeat_label.repeat_label import RepeatLabelAgent
+from parlai.core.agents import create_agent
+from parlai.core.params import ParlaiParser
+from parlai.core.worlds import create_task
+from parlai.utils.bpe import Gpt2BpeHelper
+
+
+@testing_utils.skipUnlessGPU
```

Comment:
erm, why would you insist on a gpu for this test? Seems like the opposite based on comments

Reply:
Because we only load fairseq for GPU tests :/ The alternative is to load fairseq for *all* of the tests

Reply:
Just changed now to use Kurt's use skip-unles-fairseq decorator